src/js/actions/bindChannel/parseData.ts

```diff
@@ -47,7 +49,7 @@ function parseAggregate(dispatch, state, parsed, summary) {
   if (!aggId) {
     // TODO: What about if a previous parsed.map.data.summary exists? How do
     // we derive a new agg DS to preserve transforms.
-    dispatch(aggregatePipeline(plId, aggregate));
+    dispatch(aggregatePipeline(plId, aggregate) as any); // TODO(rn) fix this so that aggregate actually returns AnyAction
```

Comment:
about your TODO -- it's as intended. `aggregatePipeline` returns a function because it is a thunk action https://github.com/reduxjs/redux-thunk